pkg/config/config_test.go

```diff
@@ -1724,6 +1726,28 @@ func TestPatchesApplyToAllObjects(t *testing.T) {
 	}
 }
 
+func GetConfig(fileName string) (cfg OperatorConfig) {
+	file, err := os.Open(fileName)
+	if err != nil {
+		panic(err)
```

Comment:
nit: could have passed `testing.T`, call `t.Helper()` and use `require.NoError(t, err)`